main.ts

```diff
@@ -52,4 +63,4 @@ bluetooth.onUartDataReceived(serial.delimiters(Delimiters.NewLine), function() {
  * ON START
  */
 bluetooth.startUartService();
-basic.showString("R")  // "R" for "ready"
+basic.showString("R"); // "R" for "ready"
```

Comment:
It can be in another PR but should we setup a linter like [tslint](https://palantir.github.io/tslint/)?

Reply:
Definitely! I'll add it in this PR.

Reply:
Added, and fixed up some other lint errors. VSCode has a nice tslint extension, too.